code/modules/jobs/job_types/bos.dm

```diff
@@ -411,23 +415,20 @@ Star Paladin
 /datum/outfit/loadout/spaladina
 	name = "Senior Assault Paladin"
 	backpack_contents = list(
-		/obj/item/gun/energy/laser/aer12=1,
-		/obj/item/stock_parts/cell/ammo/mfc=3,
-		/obj/item/grenade/flashbang=2,
-		/obj/item/grenade/chem_grenade/teargas=2,
-		/obj/item/reagent_containers/spray/pepper=1,
-		/obj/item/shield/riot/tele=1,
+		/obj/item/melee/powerfist=1,
+		/obj/item/book/granter/trait/big_leagues=1,
 		/obj/item/melee/classic_baton/telescopic=1
 		)
 
 /datum/outfit/loadout/spaladinc
 	name = "Senior Heavy Paladin"
 	backpack_contents = list(
+		/obj/item/gun/energy/laser/aer12=1,
+		/obj/item/stock_parts/cell/ammo/mfc=3,
+		/obj/item/grenade/flashbang=2,
```

Comment:
Not really sure how I feel about these loadouts. The bug with Big Leagues was fixed recently and it does **much** less damage than it used to. There's really no reason you should ever choose a powerfist or sledge over an AER12 or AER14.

Reply:
This also gives the ranged loadout **even more** grenades, which I don't think it is the way to go.

Reply:
I can remove the grenades shortly. I copied ones out. No one uses the grenadesa s is so - easy removal.

Reply:
Fiiixed